src/drive/web/modules/public/LightFileList.jsx

```diff
@@ -19,14 +22,20 @@ const LightFileList = ({
       onFolderOpen={onFolderOpen}
       withSelectionCheckbox={true}
       fileActions={actions}
+      isRenaming={isRenaming}
+      renamingFile={renamingFile}
       {...fileListProps}
     />
   </>
 )
 
 export default withRouter(
   connect(
-    mapStateToProps,
+    state => ({
+      ...mapStateToProps,
+      isRenaming: isRenaming(state),
+      renamingFile: getRenamingFile(state)
+    }),
```

Comment:
We already those props in the mapStateToProps https://github.com/cozy/cozy-drive/blob/01748efdf7a932959ac6b4636414cbabd924c154/src/drive/web/modules/public/Actions.jsx#L20

Reply:
Ah yes thanks, good catch. I know realize  they shouldn't be in `Actions` since they aren't used by any of the actions anyway.